addons/payment/models/payment_method.py

```diff
@@ -78,6 +78,11 @@ class PaymentMethod(models.Model):
         required=True,
         default='none',
     )
+    support_manual_capture = fields.Boolean(
```

Comment:
Let's move the field after `support_express_checkout` to ease maintenance (same as for providers).

Reply:
Same order in XML!